tests/unit/test_config.py

```diff
@@ -1439,3 +1448,56 @@ def test_check_for_owner(runtime_cfg):
     else:
         # Assert function doesn't raise if no owner set:
         assert WorkflowConfig.check_for_owner(runtime_cfg) is None
+
+
+@pytest.fixture(scope='module')
+def awe_config(mod_tmp_flow_config: Callable) -> WorkflowConfig:
```

Comment:
what does "awe" stand for/mean?

Reply:
Sorry, [video game reference](https://en.m.wikipedia.org/wiki/Control_(video_game)). Just means the name of the example workflow is "awe", much like we have "one" in the integration tests